compiler/qlc.ml

```diff
@@ -0,0 +1,12 @@
+type action = Raw | Ast | Semantic | Semantic_to_jast | Code_generator
```

Comment:
Oh yeah, nah it's just a basic thing - Edwards suggested we have `qlc` for compile and `ql` for running the code, similar to the convention Java uses. Previously it was `ql` and `run_ql` - I just renamed the files to match the new naming